ci/scripts/check_content.sh

```diff
@@ -6,7 +6,7 @@ set -o pipefail
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 
-if jar tvf $DIR/../../target/snowflake-jdbc.jar  | awk '{print $8}' | grep -v -E "^(net|com)/snowflake" | grep -v -E "(com|net)/\$" | grep -v -E "^META-INF" | grep -v -E "^mozilla" | grep -v -E "^com/sun/jna" | grep -v com/sun/ | grep -v mime.types; then
+if jar tvf $DIR/../../target/snowflake-jdbc.jar  | awk '{print $8}' | grep -v -E "^(net|com)/snowflake" | grep -v -E "(com|net)/\$" | grep -v -E "^META-INF" | grep -v -E "^mozilla" | grep -v -E "^com/sun/jna" | grep -v -E "^org/slf4j" | grep -v -E "^org" | grep -v com/sun/ | grep -v mime.types; then
```

Comment:
1. Why grepping against both patterns? I suppose that `^org` already covers `^org/slf4j`?
2. By the way - is excluding whole `org` not too wide?
3. If you need only `^` you don't need to specify `-E` (extended regex syntax).